theme/n21/views/lib/extended_footer.hbs

```diff
@@ -4,8 +4,8 @@
 			<div  class="col-md-4">
 				<h4>{{$t "global.link.contact" }}</h4>
 				<ul>
-					<li><a href="mailto:feedback@hpi-schul-cloud.de" target="_blank" rel="noopener" aria-label="Support-Anfragen {{$t "global.link.openInNewTabWarning"}}">Support-Anfragen:  <span class="text-nowrap">feedback@hpi-schul-cloud.de</span></a></li>
-					<li><a href="mailto:nbc-support@netz-21.de" target="_blank" rel="noopener" aria-label="Allgemeine Anfragen {{$t "global.link.openInNewTabWarning"}}">Allgemeine Anfragen: <span class="text-nowrap">nbc-support@netz-21.de</span></a></li>
+					<li><a href="mailto:nbc-wunsch@netz-21.de" target="_blank" rel="noopener" aria-label="Support-Anfragen {{$t "global.link.openInNewTabWarning"}}">Feedback/Wünsche:  <span class="text-nowrap">nbc-wunsch@netz-21.de</span></a></li>
```

Comment:
shouldn't it be nbc-support@netz-21.de ?

Reply:
This email is below